### ISSUE TITLE
Change directory Structure

### ISSUE THREAD

**Issue by oversoul** [nouraellm/Lighty#14]
How it is now, it's slightly confusing IMHO.

`app` contains the core code. usually, `app` directory is the code for the application.

i recommend renaming the `app` directory. and create a new app directory and put inside it `controllers` and `models`.

**Comment by nouraellm** [comment]
Yeah I agree, I'm already putting in mind but I'll need to find a modern/easy structure!

**Comment by oversoul** [comment]
i propose to have something simple for now. the modern/easy is to create a package for the framework.
and use composer to pull it into your app.

**Comment by nouraellm** [comment]
Well, you can leave this for me :D but it's not gonna be anytime soon cs :airplane:

**Comment by oversoul** [comment]
sure, GLHF.
